package.json

```diff
@@ -32,19 +32,20 @@
     ]
   },
   "dependencies": {
-    "eth-sig-util": "2.5.2",
-    "ethereumjs-tx": "~2.1.2",
-    "ethereumjs-util": "~6.2.1",
     "@openzeppelin/contracts": "~3.2.0",
     "@truffle/contract": "4.2.23",
     "@truffle/contract-schema": "3.3.2",
     "@truffle/hdwallet-provider": "1.6.0",
+    "eth-sig-util": "2.5.2",
+    "ethereumjs-tx": "~2.1.2",
```

Comment:
archived and replaced by `@ethereumjs/tx`

Reply:
I think we could review this and the other dependencies in the related task